tools/send-to-caas/send-utils.js

```diff
@@ -253,7 +253,14 @@ const getDateProp = (dateStr, errorMsg) => {
 
 const getOrigin = () => {
   const origin = getConfig().project || getConfig().repo;
-  if (origin) return origin;
+  const mappings = {
+    cc: 'hawks',
```

Comment:
I see additional mappings in the JIRA ticket but not in the code. 
One other thing: you should convert `mappings` to UPPERCASE and move it to the top.